frontend/src/common/hooks/useCategoryFilter/common/multiPanelOntologyUtils.ts

```diff
@@ -331,8 +331,13 @@ function buildOntologyPanelCategoryViews(
     // If panel is filtered (that is, there are selected values in this panel's parents), only include values in this
     // panel that are descendants of the selected parent values. If there are no selected values in parent panels,
     // include all values in this panel.
+
+    // Get all UI Nodes configured for this panel.
```

Comment:
thanks for all the informational comments!

Reply:
Thank you for reading them!